xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/LiveTableElement.java

```diff
@@ -304,13 +306,73 @@ public void sortBy(String columnTitle)
         getDriver().executeJavascript("return $('" + StringEscapeUtils.escapeEcmaScript(livetableId)
             + "-ajax-loader').removeClassName('hidden')");
 
-        String xpath = String.format("//table[@id = '%s']//th[contains(@class, 'xwiki-livetable-display-header-text')"
-            + " and contains(@class, 'sortable') and normalize-space(.) = '%s']", this.livetableId, columnTitle);
-        getDriver().findElement(By.xpath(xpath)).click();
+        getHeaderByColumnTitle(columnTitle).click();
 
         waitUntilReady();
     }
 
+    /**
+     * Sorts the live table on the specified column, by ascending order.
+     *
+     * @param columnTitle the column to sort on
+     * @since 13.3RC1
+     * @since 12.10.6
+     */
+    @Unstable
```

Comment:
So according to the discussions we had it shouldn't be marked as unstable.

Reply:
Good catch, thanks. Fixed